core/domain/stats_domain.py

```diff
@@ -560,23 +587,31 @@ def __init__(self, calculation_output_type):
 
 class AnswerFrequencyList(AnswerCalculationOutput):
     """Domain object that represents a list of AnswerOccurrences."""
-    def __init__(self, answer_occurrences=None):
+    def __init__(self, answer_occurrences=None, limit=None):
```

Comment:
Also, in general, why not force callers to pass in [] directly instead of having a default arg?

Reply:
Done.